This file is part of MXE.
See index.html for further information.

Commits backported (cherry-picked) from upstream.
From 9348978f78c6feae6e9bab4499ab035c6e5e25ea Mon Sep 17 00:00:00 2001
From: Mark Brand <mabrand@mabrand.nl>
Date: Sun, 27 May 2012 22:00:19 +0200
Subject: [PATCH] force static for convenience

---
 pthread.h |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/pthread.h b/pthread.h
index b4072f7..c88c886 100755
--- a/pthread.h
+++ b/pthread.h
@@ -547,6 +547,10 @@ extern "C"
  * do NOT define PTW32_BUILD, and then the variables/functions will
  * be imported correctly.
  */
+
+// MXE: Set this for convenience here since MXE only does static.
+#define PTW32_STATIC_LIB
+
 #if !defined(PTW32_STATIC_LIB)
 #  if defined(PTW32_BUILD)
 #    define PTW32_DLLPORT __declspec (dllexport)
-- 
1.7.9.2

